src/main/java/com/palettee/portfolio/repository/PortFolioRepository.java

```diff
@@ -1,7 +1,11 @@
 package com.palettee.portfolio.repository;
 
-import com.palettee.portfolio.domain.PortFolio;
-import org.springframework.data.jpa.repository.JpaRepository;
+import com.palettee.portfolio.domain.*;
+import org.springframework.data.jpa.repository.*;
 
 public interface PortFolioRepository extends JpaRepository<PortFolio, Long>, PortFolioRepositoryCustom {
+
+    @Modifying
```

Comment:
Modifing 하고 해당 영속성 콘텐츠를 한번 비워주는게 좋습니다!!
벌크 쿼리 같은경우 영속성 콘텐츠를 거치지 않고 바로 db에다가 삭제를 때려서  한번 영속성 콘텐츠를 비우는걸 추천드립니다!
```suggestion
   @Modifying(clearautomatically = true)
```

Reply:
https://github.com/prgrms-web-devcourse-final-project/WEB1_1_ZeroOne_BE/pull/39/commits/69b47788f38d1860d4afcc17e0b7b1b90bfa473d 로 추가하였습니다!